var/spack/repos/builtin/packages/camellia/package.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Camellia(CMakePackage):
+    """Camellia: user-friendly MPI-parallel adaptive finite element package,
+       with support for DPG and other hybrid methods, built atop Trilinos.
+    """
+
+    homepage = "https://bitbucket.org/nateroberts/Camellia"
+    url      = "https://bitbucket.org/nateroberts/camellia.git"
+
+    maintainers = ['nvrober-sandia']
+    
+    variant('moab', default=True,
+            description='Compile with MOAB support for reading standard mesh formats')
+
+    version('master',
+            git='https://bitbucket.org/nateroberts/camellia.git', branch='master')
+
+    depends_on('trilinos~shared+amesos+amesos2+anasazi+belos+epetra+epetraext+exodus+ifpack+ifpack2+intrepid+intrepid2+kokkos+ml+muelu+sacado+shards+teuchos+tpetra+zoltan+mumps+superlu-dist+hdf5+zlib+pnetcdf@master,12.12.1:')
+    depends_on('moab@4.9.2', when='+moab')
```

Comment:
This is the only version of moab that works?

Reply:
In my testing, 5.0.0 breaks my Mac build, so 4.9.2 is the most recent version that works.  5.0.0 does something weird to the link line.  (It looks like it somehow manages to transform ```-framework Accelerate``` into something involving ```-lAccerelate```, which breaks things.)

Reply:
Any idea if older versions will work? In that case, `moab@:4` might be sufficient.

Reply:
I imagine so.  Happy to try it.

Reply:
Anyway, it's your software, so don't let me force you to package it one way or another. But ideally, the dependency will list the exact range of versions that are known to work.